docking_demo.py

```diff
@@ -445,11 +446,13 @@ def main(exposure):
         description='This application provides a demonstration of using visual servoing to autonomously dock with the official Hello Robot docking station.')
 
     parser.add_argument('-e', '--exposure', action='store', type=str, default='auto', help=f'Set the D435 exposure to {dh.exposure_keywords} or an integer in the range {dh.exposure_range}') 
-    
+    parser.add_argument('-n', '--noviz', nargs='?', default=False, const=True)
```

Comment:
It would be good to add documentation here to clarify the meaning of this argument when someone uses --help.